content/docs/deployment/csmoperator/_index.md

```diff
@@ -0,0 +1,253 @@
+>NOTE: Dell CSM Operator will install to the 'test-csm-operator-olm' namespace by default.
+
+{{< imgproc install_olm.jpg Resize "2500x" >}}{{< /imgproc >}}
+
+3. Once installation completes, run the command `kubectl get pods -n test-csm-operator-olm` to validate the installation. If completed successfully, 
+   you should be able to see the operator-related pods in the `test-csm-operator-olm` namespace. Also, you can see a `dell-csm-operator` CSV with the status 'Succeeded'
+   created in the namespace `test-csm-operator-olm`.
+   
+{{< imgproc install_olm_pods.jpg Resize "2500x" >}}{{< /imgproc >}}
+
+### Installation Using Operator Lifecycle Manager
+`dell-csm-operator` can be installed using Operator Lifecycle Manager (OLM) on upstream Kubernetes clusters & Red Hat OpenShift Clusters.  
+The installation process involves the creation of a `Subscription` object either via the _OperatorHub_ UI or using `kubectl/oc`. While creating the `Subscription` you can set the Approval strategy for the `InstallPlan` for the Operator to - 
+* _Automatic_ - If you want the Operator to be automatically installed or upgraded (once an upgrade becomes available)
+* _Manual_ - If you want a Cluster Administrator to manually review and approve the `InstallPlan` for installation/upgrades
+
+**NOTE**: The recommended version of OLM for upstream Kubernetes is **`v0.18.2`**.
+
+##### Upstream Kubernetes
+- For installing via OperatorHub.io on Kubernetes, go to the [OperatorHub page](../../partners/operator/).
+##### Red Hat OpenShift Clusters
+- For installing via OpenShift with the Operator, go to the [OpenShift page](../../partners/redhat/).
+
+### Upgrade
+To upgrade an existing CSM operator, run `bash scripts/install.sh --upgrade`. This will upgrade existing CRDs and Operator objects.
+Manual upgrade is not supported for Operator installed in OLM environment.
+
+{{< imgproc upgrade.jpg Resize "2500x" >}}{{< /imgproc >}}
+
+### Uninstall
+#### Operator Uninstallation on an environment without OLM
+To uninstall a CSM operator, run `bash scripts/uninstall.sh`. This will uninstall the operator in `dell-csm-operator` namespace.
+
+{{< imgproc uninstall.jpg Resize "2500x" >}}{{< /imgproc >}}
+
+#### Operator Uninstallation on an environment with OLM
+To uninstall a CSM operator installed with OLM run `bash scripts/uninstall_olm.sh`. This will uninstall the operator in  `test-csm-operator-olm` namespace.
+
+{{< imgproc uninstall_olm.jpg Resize "2500x" >}}{{< /imgproc >}}
+
+## Custom Resource Definitions
+As part of the Dell CSM Operator installation, a CRD representing configuration for the CSI Driver and CSM Modules is also installed.  
+CRD which is installed in API Group `storage.dell.com`
+* containerstoragemodule
+
+For installation of the supported drivers and modules, a `CustomResource` has to be created in your cluster.
+
+## Supported CSI Drivers
+
+| CSI Driver         | Version   | ConfigVersion  |
```

Comment:
I think supported Driver version should be at the top, along with Platforms.

Reply:
Done